tests/Auth/AuthMiddlewareTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Tests\Auth;
+
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Nyholm\Psr7\ServerRequest;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Yiisoft\Yii\Web\Auth\AuthInterface;
+use Yiisoft\Yii\Web\Auth\AuthMiddleware;
+use PHPUnit\Framework\TestCase;
+use Yiisoft\Yii\Web\User\IdentityInterface;
+
+class AuthMiddlewareTest extends TestCase
+{
+    /** @var ResponseFactoryInterface */
+    private $responseFactory;
+
+    /** @var AuthInterface */
```

Comment:
Should be like this:
```

 /**
  * @var AuthInterface|PHPUnit_Framework_MockObject_MockObject
  */

```

Reply:
It is deprecated, so I change with actual class 

https://github.com/sebastianbergmann/phpunit/blob/fdc3245bc472bb1e23168b10aa0bd18716a58106/src/Framework/MockObject/MockObject.php#L21